dali/test/python/decoder/test_video.py

```diff
@@ -240,3 +241,360 @@ def test_pipeline():
     p = test_pipeline(batch_size=batch_size, num_threads=1, device_id=0)
 
     assert_raises(RuntimeError, p.run)
+
+
+cfr_files = [
```

Comment:
Do you know how much these new tests take (and if we should trim them/make shorter/make optional with `slow` atribute)?

Reply:
they are quite quick to run (few seconds for the whole suite on my machine)